src/nightwatch/nightwatch.config.js

```diff
@@ -0,0 +1,80 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "chromedriver" }] */
+
+import chromedriver from 'chromedriver';
+import webpack from 'webpack';
+import WebpackDevServer from 'webpack-dev-server';
+import seleniumServerStandaloneJar from 'selenium-server-standalone-jar';
+
+let port = 8080;
+
+const nightwatchConfig = (webpackConfig, srcFolders, providedPort) => {
+  if (providedPort) {
+    port = providedPort;
+  }
+
+  const webpackServer = new WebpackDevServer(webpack(webpackConfig), { quiet: true, hot: false, inline: false });
+  const stopWebpackDevServer = done => webpackServer.close(() => done());
+  const startWebpackDevServer = done => webpackServer.listen(port, '0.0.0.0', () => done());
```

Comment:
I will take a look at this package. I think this would be valuable to include. Currently if a port is busy, nightwatch ends up hanging because webpack does not return any information when a listen error occurs, so we cannot close the server.